scripts/populate_expressions_list.py

```diff
@@ -0,0 +1,216 @@
+from os import path, walk
+from sys import argv
+import json
+import re
+
+# To use this script clone the QGIS/QGIS repository, and checkout the release
+# and run:
+#
+# python3 populate_expression.py [path/to/qgis/repository]
+#
+# If no argument is given, the script assumes that the QGIS repository is in the
+# same folder as the QGIS-Documentation repository
+
+# Determine the folder where the json files are
+if len(argv) > 1:
+    qgis_repo_path = argv[1]
+else:
+    qgis_repo_path = path.join(path.dirname(__file__), '..', '..', 'QGIS')
+help_folder = path.join(qgis_repo_path, 'resources/function_help/json')
+# expression help folder
+output_folder = path.join(path.dirname(__file__), '..', 'docs/user_manual/working_with_vector/expression_help')
+
+def sphynxify_html(text, base_indent=0):
+    filler = base_indent * ' '
+
```

Comment:
```suggestion

    text = text.replace('"','\'')
```
There are some double quotes in format_date that makes the HTML build fail on my master branch. Could be fixed upstream or like this.

Reply:
@havatv I think this is already fixed in the upstream PR. See here:

https://github.com/qgis/QGIS/pull/37837/files#diff-31fdd1cc9f66c0b085723bdf6d260aa9R8

Reply:
@havatv do you think we need to do commit this change anyway, just in case?

Reply:
No need to commit if we are sure that upstream will stop using double quotes...
But the change shouldn't do any harm if committed.